scripts/collectionIndex.cdc

```diff
@@ -0,0 +1,33 @@
+import NonFungibleToken from "../contracts/standard/NonFungibleToken.cdc"
+import MetadataViews from "../contracts/standard/MetadataViews.cdc"
+import NFTRegistry from "../contracts/NFTRegistry.cdc"
+import FindViews from "../contracts/FindViews.cdc"
+
+pub struct MetadataCollections {
+
+	pub let publicPath: PublicPath 
```

Comment:
We have to think if we want to expose strings or PublicPath types. I am not sure if creating paths is possible in cadut/fcl so I think sending out a string here and accepting a string in the resolvePartialCollection might make sense.